### PR TITLE
Upgrade to latest github-pages and enable Solidity syntax highlighting

### PR DESCRIPTION
**Pull request by axic**
After https://github.com/github/pages-gem/pull/683 / https://github.com/github/pages-gem/pull/690 was merged this is possible now.

This allows EIPs to use solidity syntax highlighting. Currently there are 330 instances of it:

```
$ grep "\`solidity" * | wc -l
     330
```

And some EIPs used `typescript` or `javascript` as an alternative, because that was highlighted already:
```
$ grep "\`typescript" * | wc -l
      16
$ grep "\`javascript" * | wc -l
      25
```

Thanks @veox again in adding highlighting to rouge.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - File .travis.yml is not an EIP
 - File Gemfile is not an EIP
 - File Gemfile.lock is not an EIP
 - File _config.yml is not an EIP
 - EIP 2309 requires approval from one of (@pizza-r0b)

**Comment by axic** [comment]
This should be merged after review, cc @Souptacular @gcolvin @nicksavers @Arachnid

**Comment by MicahZoltu** [comment]
@axic This now has conflicts and cannot be merged.  Seeing as how it has been 1.5 months since you opened this and no one has commented, I would propose we just merge it and if things break we can fix it with a follow-up PR.  Worst case scenario if things go south isn't *that* bad, and I think having this sit open for 6 months while we wait for all of the editors to have time to thoroughly review it isn't ideal.

It *would* be nice to get a review from at least one person that understands Ruby/Jekyll just to make sure things are sane, but otherwise I don't think it is critical to wait for total consensus.

> And some EIPs used typescript or javascript as an alternative, because that was highlighted already:

Some of these may be actual TS or JS, so we should take care not to automatically convert all of them to solidity automatically when the time comes.

**Comment by axic** [comment]
> > And some EIPs used typescript or javascript as an alternative, because that was highlighted already:
>
> Some of these may be actual TS or JS, so we should take care not to automatically convert all of them to solidity automatically when the time comes.

I have manually checked and changed the appropriate ones -- did not use any regex to do this.

File: EIPS/eip-2309.md [hunk 1]
```diff
@@ -121,8 +121,6 @@ This extension was written to allow for the smallest change possible to the orig
 
 [Cargo](https://cargo.build)
 
-<!-- ## Implementation
```
Comment (axic) [inline EIPS/eip-2309.md:124]: Semi-invalid HTML right here.

**Pull request merged**